packages/meta-harness/lib/proposer.py

```
@@ -309,13 +309,32 @@ def render_proposer_prompt(
     max_overlay_bytes = proposer_cfg.get("max_overlay_bytes", DEFAULT_MAX_OVERLAY_BYTES)
     rendered_focus_runs = _format_focus_runs(focus_run_id, focus_run_ids)
     rendered_valid_runs = _join_or_none([str(run_id) for run_id in valid_based_on_run_ids or ()])
-    if target.startswith("skill:"):
+    if target == "routing-config":
+        baseline_input_hint = (
+            "- routing config の現在値: 下記の変更メニュー（読み取り専用 context）"
```

Ruff (0.15.21)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)